### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Oskar2301**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { WeatherAppComponent } from './pages/weather-app/weather-app.component';
+
+export const routes: Routes = [
+  { path: '', pathMatch: 'prefix', redirectTo: 'home' },
```
Comment (Oskar2301) [inline src/app/app-routing.module.ts:6]: Move path in enum

File: src/app/app.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterOutlet } from '@angular/router';
+import { ThemeService } from './shared/services/theme.service';
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [CommonModule, RouterOutlet],
```
Comment (Oskar2301) [inline src/app/app.component.ts:9]: delete this imporst and standalone

File: src/app/app.config.server.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { mergeApplicationConfig, ApplicationConfig } from '@angular/core';
```
Comment (Oskar2301) [inline src/app/app.config.server.ts:1]: For what this file? If it not need just delete

File: src/app/app.config.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { ApplicationConfig } from '@angular/core';
```
Comment (Oskar2301) [inline src/app/app.config.ts:1]: the same

File: src/app/pages/weather-app/weather-app.component.ts [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  public flower = 'assets/flower.png';
+  public leaf = 'assets/leaf.png';
+  public cloudLighting = 'assets/cloud-lighting.png';
+  public cloudRain = 'assets/cloud-rain.png';
+  public cloudLightingSun = 'assets/cloud-lighting-sun.png';
+  public cloudSunny2 = 'assets/cloud-sun-2.png';
+
+  constructor(public themeService: ThemeService) {
+    this.updateThemeImage();
+    this.themeService.themeChanged.subscribe(() => {
```
Comment (Oskar2301) [inline src/app/pages/weather-app/weather-app.component.ts:28]: for what this subscribe?

File: src/app/shared/services/theme.service.ts [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { ThemeEnum } from '../../utilities/enums/theme.enum';
+import { Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ThemeService {
+  public storedTheme = localStorage.getItem('theme-color');
+  public themeChanged = new Subject<void>();
```
Comment (Oskar2301) [inline src/app/shared/services/theme.service.ts:10]: for what this subject? delete him

File: src/app/pages/weather-app/weather-app.component.ts [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+  public snowflake = 'assets/snowflake.png';
+  public flower = 'assets/flower.png';
+  public leaf = 'assets/leaf.png';
+  public cloudLighting = 'assets/cloud-lighting.png';
+  public cloudRain = 'assets/cloud-rain.png';
+  public cloudLightingSun = 'assets/cloud-lighting-sun.png';
+  public cloudSunny2 = 'assets/cloud-sun-2.png';
+
+  constructor(public themeService: ThemeService) {
+    this.updateThemeImage();
```
Comment (Oskar2301) [inline src/app/pages/weather-app/weather-app.component.ts:27]: move this.updateThemeImage(); in ngOnInit

File: src/app/pages/weather-app/weather-app.component.ts [hunk 8]
```diff
@@ -0,0 +1,45 @@
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ThemeService } from '../../shared/services/theme.service';
+import { ThemeEnum } from 'src/app/utilities/enums/theme.enum';
+
+@Component({
+  selector: 'app-weather-app',
+  standalone: true,
+  imports: [CommonModule],
```
Comment (Oskar2301) [inline src/app/pages/weather-app/weather-app.component.ts:9]: delete imports and standalone

File: src/app/pages/weather-app/weather-app.component.ts [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+  public themeImage = '';
+  public cloudSunny = 'assets/cloud-sun.svg';
+  public darkIcon = 'assets/moon.png';
+  public snowflake = 'assets/snowflake.png';
+  public flower = 'assets/flower.png';
+  public leaf = 'assets/leaf.png';
+  public cloudLighting = 'assets/cloud-lighting.png';
+  public cloudRain = 'assets/cloud-rain.png';
+  public cloudLightingSun = 'assets/cloud-lighting-sun.png';
+  public cloudSunny2 = 'assets/cloud-sun-2.png';
```
Comment (Oskar2301) [inline src/app/pages/weather-app/weather-app.component.ts:24]: cloudSunny2? rename variable

File: src/app/pages/weather-app/weather-app.component.scss [hunk 10]
```diff
@@ -0,0 +1,628 @@
+@import "src/app/styles/theme";
+
+.weather {
+  font-family: Roboto Slab, serif;
+  background: $background-color;
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  padding: 80px 120px;
```
Comment (Oskar2301) [inline src/app/pages/weather-app/weather-app.component.scss:10]: space

File: src/app/pages/weather-app/weather-app.component.scss [hunk 11]
```diff
@@ -0,0 +1,628 @@
...
+    object-fit: cover;
+    width: 40px;
+  }
+
+  &__title {
+    font-size: 40px;
+    font-weight: 700;
+    color: $title-color-text;
+  }
+  &__theme {
```
Comment (Oskar2301) [inline src/app/pages/weather-app/weather-app.component.scss:48]: space

File: src/app/pages/weather-app/weather-app.component.scss [hunk 12]
```diff
@@ -0,0 +1,628 @@
...
+    border: none;
+    cursor: pointer;
+    margin-left: auto;
+    background: transparent;
+    &_image {
```
Comment (Oskar2301) [inline src/app/pages/weather-app/weather-app.component.scss:53]: space

File: src/app/pages/weather-app/weather-app.component.scss [hunk 13]
```diff
@@ -0,0 +1,628 @@
...
+
+  &__main {
+    position: relative;
+    font-size: 70px;
+    font-weight: 700;
+    flex-direction: column;
+    justify-content: center;
+    display: flex;
+    color: $title-color-text;
+    gap: 0;
```
Comment (Oskar2301) [inline src/app/pages/weather-app/weather-app.component.scss:76]: space, and for what gap 0?

File: src/app/pages/weather-app/weather-app.component.scss [hunk 14]
```diff
@@ -0,0 +1,628 @@
...
+  }
+
+  &__flower {
+    position: absolute;
+    width: 50px;
+    top: 50%;
+    left: 99.4%;
+    filter: blur(2px);
+  }
+  &__bottom {
```
Comment (Oskar2301) [inline src/app/pages/weather-app/weather-app.component.scss:95]: space

File: src/app/pages/weather-app/weather-app.component.scss [hunk 15]
```diff
@@ -0,0 +1,628 @@
...
+    position: relative;
+  }
+  &__text {
```
Comment (Oskar2301) [inline src/app/pages/weather-app/weather-app.component.scss:98]: space

File: src/app/app.component.ts (hunk 2 above)
Comment (plopdark) [reply src/app/app.component.ts:9]: Without that my app is broking

File: src/app/app.config.ts (hunk 4 above)
Comment (plopdark) [reply src/app/app.config.ts:1]: These files were created when i just create a project.When i said that i don't have app module they were there.I tried to delete it but app was broken.Maube it's beacuse i launched this app on angular 17

**Pull request closed without merging**